### ISSUE TITLE
Can you publish a release version to convert apng into webp like on this repository?

### ISSUE THREAD

**Issue by FurkanGozukara** [sizeofint/webp-animation#1]
https://github.com/Benny-/apng2webp/releases

**Comment by sizeofint** [comment]
Hi, sorry for late answer just saw this issue, please use this package: https://github.com/sizeofint/webpanimation this is current and active project.
I made some example, you may find interesting: https://github.com/sizeofint/webpanimation/tree/apng-to-webp/examples/apng-to-webp

**Comment by FurkanGozukara** [comment]
apng and webp version of the elephant looks different any reason?

**Comment by sizeofint** [comment]
Yeah, if look carefully, I see webp version has more dark contours than apng version in elephant example...
Currently I'm not sure, but this might be reason of not correctly implementing dispose, blending specifications of apng, anyway in future I will try to figure out, meanwhile if someone has suggestions, any contributions are welcome
